doc/develop/getting_started/index.rst

```diff
@@ -686,6 +689,7 @@ Build the :ref:`blinky-sample` with :ref:`west build <west-building>`, changing
 
 The ``-p auto`` option automatically cleans byproducts from a previous build
 if necessary, which is useful if you try building another sample.
+The ``-p always`` option may also be used to force west to create a clean build.
```

Comment:
We should probably mention `-p` (always pristine) in the getting started, it's probably more deterministic (?)

```
pristine builds:
  A "pristine" build directory is empty. The -p option controls
  whether the build directory is made pristine before the build
  is done. A bare '--pristine' with no value is the same as
  --pristine=always. Setting --pristine=auto uses heuristics to
  guess if a pristine build may be necessary.

  -p [{auto,always,never}], --pristine [{auto,always,never}]
                        pristine build folder setting
```

Reply:
I've moved the default instructions to use `-p always`, and made a note about `-p auto` so that users are aware of it.